pkg/server/server.go

```diff
@@ -801,7 +801,7 @@ func validateAdmin(imageSpace string, w http.ResponseWriter, r *http.Request) bo
 	if err != nil {
 		message := fmt.Sprintf("Unable to get permission token %s", err)
 		kiln.LogError.Printf(message)
-		writeErrorResponse(http.StatusUnauthorized, message, w)
```

Comment:
What we could do is just update `30x/authsdk` to return the status code instead of an error, or create an error object that contains the original status code, and then consult that from the consumers.  That would allow the caller of `#IsOrgAdmin` the ability to be more aware as to what the underlying error is instead of just being given a generic error.  Thoughts?